### DISCUSSION TITLE
I can't launch Karavan anywhere

### DISCUSSION THREAD

**Discussion by carlosdelgadoEAG** [apache/camel-karavan#1143] [Q&A]
I have followed multiple times the steps to start Karavan in docker and I only get to see the web app. Once I try to design a simple route (timer + log as you can see in an example youtube video) it automatically fails to launch the test. 

I installed docker, java, mvn and followed every single step in different environments. 

Errors I get: 

- java.net.UnknownHostException (on terminal)
- [ERROR] Could not read artifact descriptor for org.apache.camel:camel-bom:pom:4.4.0
- Caused by: java.nio.file.AccessDeniedException: /karavan/.m2/org

Anything I can be missing?

**Comment by mgubaidullin** [comment]
Karavan 4.4.0 is not yet released. Please try 4.3.0 https://github.com/apache/camel-karavan/blob/4.3.0/docs/WEB_DOCKER.md

**Comment by davsclaus** [comment]
4.4.0 has been released, try this and report back if still some problems
